### PR TITLE
Update app list UI

### LINKED ISSUE #524: [Portal] Update app list UI
# App list

Screenshot 2020-10-16 at 11 30 12 PM

# Create app

Screenshot 2020-10-16 at 11 31 13 PM

# Screen header

Screenshot 2020-10-16 at 11 31 50 PM

### PR DESCRIPTION
**Pull request by axolotl0212**
fix #524

# Screenshots

Screenshot 2020-10-17 at 1 17 28 AM

Screenshot 2020-10-17 at 2 02 53 AM

Screenshot 2020-10-19 at 12 36 41 PM

### CODE REVIEW

File: portal/src/graphql/portal/CreateAppScreen.tsx [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
 import { useTextField } from "../../hook/useInput";
 
+import styles from "./CreateAppScreen.module.scss";
+
 interface CreateAppProps {
   isCreating: boolean;
   createApp: (appID: string) => Promise<string | null>;
 }
 
+const APP_ID_SCHEME = "http://";
```
Comment (louischan-oursky) [inline portal/src/graphql/portal/CreateAppScreen.tsx:18]: https

**Comment by axolotl0212** [comment]
@louischan-oursky rebased, fix typo and added 1 more commit to fix long app name can overflow out of app list item

**Comment by louischan-oursky** [comment]
We have merged https://github.com/authgear/authgear-server/pull/533
Can you update the PR to use the new field in the runtime config?

**Comment by axolotl0212** [comment]
@louischan-oursky updated to using suffix from runtime config

**Pull request merged**